package.json

```diff
@@ -7,8 +7,8 @@
     "scripts": {
         "build": "shx rm -rf dist/ && tsc",
         "doc": "typedoc --out \"ts-docs/$(npm run version --silent)\" src",
-        "test": "npm run build && mocha --ui bdd --recursive ./dist/test --timeout 120000 --unhandled-rejections=strict ",
-        "test:vector": "npm run build && shx cp -R ./test/test-vector ./dist/test/test-vector && mocha --ui bdd --recursive ./dist/test/vector --timeout 200000 --unhandled-rejections=strict ",
+        "test": "npm run build && mocha --ui bdd --recursive ./dist/test/unit-tests --timeout 120000 --unhandled-rejections=strict ",
+        "test:vector": "npm run build && shx cp -R ./test/test-vector ./dist/test/test-vector && mocha --ui bdd --recursive ./dist/test/vector-tests --timeout 500000 --unhandled-rejections=strict ",
```

Comment:
Is it possible to avoid the copy?

Another way to run the test is having typescript only compiling production code into the dist folder and then having the mocha tests running the test code directly with -r ts-node/register

https://github.com/symbol/symbol-bootstrap/blob/dev/package.json#L109
```
"vector": "nyc --reporter=lcov --extension .ts mocha -r ts-node/register --timeout 900000 --forbid-only \"test/**/*.vector.ts\"",
"test": "nyc --reporter=lcov --extension .ts mocha -r ts-node/register --timeout 900000 --forbid-only \"test/**/*.spect.ts\"",
````

This way, the mocha working dir and where the test ts files are located are the same. 
You don't need a separate vector-tests folder, just different suffix does the trick.